repos/fdbt-site/src/pages/defineCapPricingPerDistance.tsx

```diff
@@ -0,0 +1,207 @@
+            <CsrfForm action="/api/capPricingPerDistance" method="post" csrfToken={csrfToken}>
+                <>
+                    <ErrorSummary errors={errors} />
+                    <h1 className="govuk-heading-l" id="cap-pricing-per-distance-heading">
+                        Enter your distance cap details
+                    </h1>
+                    <div className="flex-container">
+                        <div className="govuk-!-margin-left-1 govuk-!-margin-right-2">
+                            <FormGroupWrapper
+                                errors={errors}
+                                errorIds={[`cap-pricing-per-distance-minimum-price`]}
+                                hideErrorBar
+                            >
+                                <>
+                                    <>
+                                        <label className="govuk-label" htmlFor="minimum-price">
+                                            Minimum price(£)
+                                        </label>
+
+                                        <span className="govuk-hint" id={`minimum-price-hint`}>
+                                            e.g. £2.99
+                                        </span>
+                                    </>
+                                    <div className="govuk-input__wrapper">
+                                        <div className="govuk-input__prefix" aria-hidden="true">
+                                            £
+                                        </div>
+                                        <input
+                                            className="govuk-input govuk-input--width-5"
+                                            id={`minimum-price`}
+                                            name={`minimumPrice`}
+                                            type="text"
+                                            onChange={(e) => {
+                                                setCapPricingPerDistanceData({
+                                                    ...capPricingPerDistanceData,
+                                                    minimumPrice: e.target.value,
+                                                });
+                                            }}
+                                            value={capPricingPerDistanceData.minimumPrice || ''}
+                                        />
+                                    </div>
+                                </>
+                            </FormGroupWrapper>
+                        </div>
+
+                        <div className="govuk-!-margin-left-2 govuk-!-margin-right-2">
+                            <FormGroupWrapper
+                                errors={errors}
+                                errorIds={[`cap-pricing-per-distance-maximum-price`]}
+                                hideErrorBar
+                            >
+                                <>
```

Comment:
The errors are working fine, and the href of the errors correctly links the user to the input box which errored, but there is no red highlight around erroring input boxes. See other pages for how to do this.